lib/srv/desktop/rdp/rdpclient/src/lib.rs

```diff
@@ -1980,3 +1980,10 @@ pub(crate) type Messages = Vec<Message>;
 pub(crate) trait Encode: std::fmt::Debug {
     fn encode(&self) -> RdpResult<Message>;
 }
+
+/// This is the maximum size of an RDP message which we will accept
+/// over a virtual channel.
+///
+/// Note that this is not an RDP defined value, but rather one we've chosen
+/// in order to harden system security.
+const MAX_ALLOWED_VCHAN_MSG_SIZE: usize = 2_usize.pow(21); // 2MB
```

Comment:
nitpick: I know that it is the correct value, but at first for me it was a bit confusing at how it's the 2 to the power of 21 that makes 2MB.

Maybe it would be more readable if we separate the unit with the value, e.g:
```
const MAX_ALLOWED_VCHAN_MSG_SIZE: usize = 2 * 2_usize.pow(20);
```
or write it like:

```
const MAX_ALLOWED_VCHAN_MSG_SIZE: usize = 2 << 20;
```
or
```
const MAX_ALLOWED_VCHAN_MSG_SIZE: usize = 2 * 1024 * 1024;
```
It seems like we're using the last style in the codebase:
https://github.com/gravitational/teleport/blob/master/lib/events/sizelimit.go#L29

Of course, it's up to you! 😄

Reply:
77cc8fc8dde8178d9706aaa1e23458a89f982b1d